packages/shared/src/components/ProfileMenu/ProfileSectionItem.tsx

```diff
@@ -38,13 +41,15 @@ export const ProfileSectionItem = ({
   className,
   title,
   href,
-  icon,
+  icon: Icon,
   onClick,
   external,
+  isActive,
 }: ProfileSectionItemProps): ReactElement => {
   const tag = href ? TypographyTag.Link : TypographyTag.Button;
 
-  const isExternal = href && external;
+  const showLinkIcon = href && external;
+  const openNewTab = showLinkIcon && !href.startsWith(webappUrl);
```

Comment:
since its `/` in webapp and in extension its a full URL, so it might not be that relevant, but I am fine with it